File: qualifier-week-3/graded assignments/GrPA2 - Graded.c
#include <stdio.h>
int main() {
    int timeInSeconds, hours, minutes, seconds;
    scanf("%d", &timeInSeconds);
    // Write Solution code below
     hours = timeInSeconds / 3600;         // Calculate the number of hours
     minutes = (timeInSeconds % 3600) / 60; // Calculate the number of minutes
     seconds = timeInSeconds % 60;          // Calculate the number of seconds
    printf("%d hours, %d minutes, %d seconds", hours, minutes, seconds);
    return 0;
}

File: qualifier-week-3/graded assignments/GrPA2 - Graded.go
package main

import "fmt"

func main() {
	var timeInSeconds, hours, minutes, seconds int
	fmt.Scan(&timeInSeconds)

	hours = timeInSeconds / 3600
	minutes = (timeInSeconds % 3600) / 60
	seconds = timeInSeconds % 60
	fmt.Printf("%d hours, %d minutes, %d seconds\n", hours, minutes, seconds)
}
